frontend/tests/ui/util.py

```diff
@@ -310,7 +310,8 @@ def assert_no_js_errors(
 def create_problem(
         driver,
         problem_alias: str,
-        resource_path: str = 'frontend/tests/resources/triangulos.zip'
+        resource_path: str = 'frontend/tests/resources/triangulos.zip',
```

Comment:
```suggestion
        *,
        resource_path: str = 'frontend/tests/resources/triangulos.zip',
```

así se obliga a los que llaman esta función a pasarle los parámetros por nombre.